Processing of short videos fails

**Describe the bug**
When I upload a very short video (< 1 second) via Opencast Studio the processing of the uploaded video fails.
To be more precise: The "Studio Upload" workflow fails at the "image" operation.

![Screenshot 2021-02-26 at 11 35 38](https://user-images.githubusercontent.com/4180675/109289707-e0c48d00-7826-11eb-8580-95c6244ef681.png)
![Screenshot 2021-02-26 at 11 33 02](https://user-images.githubusercontent.com/4180675/109289716-e5894100-7826-11eb-8d6d-2858a000fef0.png)


opencast.log:
```
2021-02-26T11:26:18,953 | ERROR | (AbstractJobProducer$JobRunner:343) - Error handling operation 'Image':
org.opencastproject.serviceregistry.api.ServiceRegistryException: Error handling operation 'Image'
        at org.opencastproject.composer.impl.ComposerServiceImpl.process(ComposerServiceImpl.java:1517) ~[?:?]
        at org.opencastproject.job.api.AbstractJobProducer$JobRunner.call(AbstractJobProducer.java:313) [!/:?]
        at org.opencastproject.job.api.AbstractJobProducer$JobRunner.call(AbstractJobProducer.java:272) [!/:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: org.opencastproject.composer.api.EncoderException: Extracted image /srv/opencast/workspace/mediapackage/437a4df5-52ff-42fd-b88f-f7dc53db611e/e0feb09d-e50f-4428-b33a-58d79b1bd9fa/2021_02_26_11_23___Short_Video___1sec____Johannes__presentation__OC_Studio_-bd4a44d8-7802-433f-bfab-dbd6ffef7ada_c7e54c9c-f825-45d3-8b4d-7f285987b71b-search.jpg is empty!
        at org.opencastproject.composer.impl.ComposerServiceImpl.extractImages(ComposerServiceImpl.java:1175) ~[?:?]
        at org.opencastproject.composer.impl.ComposerServiceImpl.process(ComposerServiceImpl.java:1409) ~[?:?]
        ... 6 more
2021-02-26T11:26:28,089 | ERROR | (WorkflowOperationWorker:141) - Workflow operation 'operation:'image', position:16, state:'FAILED'' failed
org.opencastproject.workflow.api.WorkflowOperationException: Image extraction failed
        at org.opencastproject.workflow.handler.composer.ImageWorkflowOperationHandler$Extractor.main(ImageWorkflowOperationHandler.java:213) ~[?:?]
        at org.opencastproject.workflow.handler.composer.ImageWorkflowOperationHandler.start(ImageWorkflowOperationHandler.java:146) ~[?:?]
        at org.opencastproject.workflow.impl.WorkflowOperationWorker.start(WorkflowOperationWorker.java:193) ~[!/:?]
        at org.opencastproject.workflow.impl.WorkflowOperationWorker.execute(WorkflowOperationWorker.java:116) [!/:?]
        at org.opencastproject.workflow.impl.WorkflowServiceImpl.runWorkflowOperation(WorkflowServiceImpl.java:801) [!/:?]
        at org.opencastproject.workflow.impl.WorkflowServiceImpl.process(WorkflowServiceImpl.java:1833) [!/:?]
        at org.opencastproject.workflow.impl.WorkflowServiceImpl$JobRunner.call(WorkflowServiceImpl.java:2283) [!/:?]
        at org.opencastproject.workflow.impl.WorkflowServiceImpl$JobRunner.call(WorkflowServiceImpl.java:2249) [!/:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**To Reproduce**

1. Go to Opencast Studio
2. Record a screencast (without audio) with less than one second duration
3. Upload the screencast
4. Wait a while for video processing and see the error on the admin panel

**Environment Information**
 - OS: Ubuntu 18.04
 - Browser Chrome
 - Opencast version 9.2 (installed from repository)

--- a/etc/encoding/engage-images.properties
+++ b/etc/encoding/engage-images.properties
@@ -34,7 +34,7 @@ profile.player-preview.http.input = visual
 profile.player-preview.http.output = image
 profile.player-preview.http.suffix = -player.jpg
 profile.player-preview.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
-  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),fps=1,reverse,scale=-2:720 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),scale=-2:720,fps=25,reverse \
   #{out.dir}/#{out.name}#{out.suffix}
 
 # Slide preview images as shown in the player
@@ -50,7 +50,7 @@ profile.search-cover.http.input = visual
 profile.search-cover.http.output = image
 profile.search-cover.http.suffix = -search.jpg
 profile.search-cover.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
-  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),fps=1,reverse,scale=160:-2 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),scale=160:-2,fps=25,reverse \
   #{out.dir}/#{out.name}#{out.suffix}
 
 profile.search-cover.http.downscale.name = Downscale to cover image for engage

--- a/etc/encoding/feed-images.properties
+++ b/etc/encoding/feed-images.properties
@@ -32,4 +32,6 @@ profile.feed-cover.http.name = cover image for feeds
 profile.feed-cover.http.input = visual
 profile.feed-cover.http.output = image
 profile.feed-cover.http.suffix = -feed.jpg
-profile.feed-cover.http.ffmpeg.command = -ss #{time} -i #{in.video.path} -r 1 -frames:v 1 -filter:v yadif,scale=-1:54 #{out.dir}/#{out.name}#{out.suffix}
+profile.feed-cover.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),scale=-1:54,fps=25,reverse \
+  #{out.dir}/#{out.name}#{out.suffix}

